### DISCUSSION TITLE
Bug：After dragging the record to a different date, the date display on the right is not updated.

### DISCUSSION THREAD

**Discussion by c2006878** [nocodb/nocodb#9725] [General]
![image](https://github.com/user-attachments/assets/b1aafb60-787a-4475-93be-405051f09301)

![image](https://github.com/user-attachments/assets/28787450-8e55-45f5-9820-1e8fa65c2e76)

After dragging the record to a different date, the date display on the right is not updated. However, after clicking, the date in the form has changed.

**Comment by dstala** [comment]
Issue created to follow up further
https://github.com/nocodb/nocodb/issues/9732
